lms/services/lti_names_roles.py

```diff
@@ -0,0 +1,56 @@
+"""
+Service to talk to the Name and Roles LTIA API
+
+https://www.imsglobal.org/spec/lti-nrps/v2p0
+"""
+from typing import List, TypedDict, Optional
+from lms.services.ltia_http import LTIAHTTPService
+
+
+Member = TypedDict(
+    "Member",
+    {
+        "email": str,
+        "family_name": str,
+        "name": str,
+        "picture": str,
+        "roles": List[str],
+        "status": str,
+        "user_id": str,
+        "lti11_legacy_user_id": Optional[str],
+    },
+)
+
+
+class LTINamesRolesServices:
```

Comment:
This file could be `services/lti/names_and_roles` but we'd still want to call the service `LTINamesRolesService`

As there's no more LTIA services I don't think we need the `service/lti/` namespace.